dts/msm8916/msm8216-samsung-r07.dts

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+    // This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <248 0>;
+	qcom,board-id = <0xF808FF01 7>;
+
+    fortuna3g {
+		model = "Samsung Galaxy Grand Prime 3G (SM-G530H)";
+		compatible = "samsung,fortuna3g", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-bootloader = "G530H*";
+
+		lk2nd,keys = <KEY_HOME 109 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+
+		samsung,muic-reset {
+			i2c-gpio-pins = <2 3>;
+			i2c-address = <0x25>;
+		};
+
+		panel {
```

Comment:
Can you make an **additional commit** where you make a `msm8916-samsung-gprime.dtsi` similar to `msm8916-wingtech-wt865x8.dtsi` and put the `lk2nd,keys`, `samsung,muic-reset`, `panel` there?

Then include it everywhere needed (see `lenovo-a6010` in `msm8916-qrd9-v1.dts` for an example):

- fortuna3g (msm8216-samsung-r07.dts)
- fortuna3g (msm8929-samsung-r04.dts)
- fortunaltezt (msm8916-samsung-r11.dts)
- gprimeltecan (msm8916-samsung-r02.dts)

I _think_ it should match for all of them.